content/download/lts.adoc

```diff
@@ -0,0 +1,128 @@
+---
+layout: simplepage
+title: LTS Release Line
+---
+:toc:
+
+The weekly Jenkins releases deliver bug fixes and new features rapidly to users and plugin developers who need them.
+But for more conservative users, it's preferable to stick to a release line which changes less often and only receives important bug fixes, even if such a release line lags behind in terms of features.
+Several companies maintain their own private branches off of Jenkins for stabilization and internal customizations.
+We encourage everybody to shift a part of the effort to this release line.
+
+This is called the Jenkins *Long-Term Support* release, or *LTS*. The concept is very similar to the link:https://wiki.ubuntu.com/LTS[LTS concepts in Ubuntu] and our model is described below.
+
+## Model
+
+Every 12 weeks, the community will pick one of the relatively recent releases by consensus and mark it as the baseline for the next "stable but older" release line.
+Let's say this was version X.
+We'll create a branch from X to produce stable but older patch releases of *X.1*, *X.2*, and *X.3*.
+Changes to this branch will be restricted to backported cherry-picked bug fixes from the trunk that are "battle-tested" — meaning those commits that have already been a part of a main line release for more than a week.
+There are 3 minor releases for a baseline published in four week cycles.
+A release candidate is published two weeks before a minor release.
+
+The following table demonstrates release dates within the 12 week cycle:
+
+++++
+<style>
+  table#process th, table#process td {
+    min-width: 60px;
+    border: 1px solid grey;
+    padding: 3px;
+  }
+</style>
+<table id="process">
```

Comment:
I tried and it's a mess. All the styles (`grid`, `frame`, etc.) get ignored, and I didn't manage to make it look in any way sane.

Whatever I did, it always looked like the bottom table here:

> ![screen shot](https://cloud.githubusercontent.com/assets/1831569/26464029/a4ea2040-4186-11e7-8a1d-aad8c5e9b661.png)

Also, there appears to be no way to have a header column (cells made up of `th`), there's only header rows.